shell/store/github.js

```diff
@@ -10,6 +18,13 @@ const fetchGithubAPI = async(endpoint) => {
   return await response.json();
 };
 
+export const getters = {
+  // classify: (state, getters, rootState) => (obj) => {
```

Comment:
These are the bits that break linting. As we decided to split out the `deploy from github commit` to be done after the re-deploy functionality is added... we just need to reference that issue number here